http1_server.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+import socket
+import email.utils
+
+
+def response_ok(request):
+    response_code = "HTTP/1.1 200 OK"
+    date = "Date: {}".format(email.utils.formatdate(usegmt=True))
+    headers = "Content-Type: Text/HTML"
+    response = "{}\r\n{}\r\n{}\r\n\r\n".format(response_code, date, headers)
```

Comment:
This is too brittle.  What happens if you want to add more headers?  A body?  Can you think of a better way to handle this?